corehq/motech/repeaters/models.py

```diff
@@ -242,11 +242,17 @@ def get_auth(self):
             return HTTPDigestAuth(self.username, self.password)
         return None
 
-    def send_request(self, repeat_record, payload):
+    @property
+    def verify(self):
+        # overwrite to allow/skip certificate verification when sending request
+        # to https urls
+        return None
```

Comment:
should this default to `True`? This seems like it will set the default to not verify SSL certificates, which seems bad.

Reply:
Passed `None` to use the default value or skip passing as argument..
https://github.com/dimagi/commcare-hq/pull/18196/files#diff-e6650b37cfd2fbc815ba0d96d2476082R36

Reply:
https://github.com/dimagi/commcare-hq/pull/18196#discussion_r144617076

Reply:
I still think this is a little confusing to have this have 3 states (`True`, `False`, and `None`) when the resulting behavior of `True` and `None` are the same. That might just be stylistic though, so up to you if you want to change it.

Reply:
Agree. Was just scared if this would start verifying certificates where not needed. 
Did a quick check and looks like a safe change so updated to `True`